templates/case-studies/index.html

```diff
@@ -6,41 +6,54 @@
-                        description="It’s not just expert personnel, satellites and spacecraft: it takes mountains of software, computer systems, data and networks to get these missions off the ground. These missions are often unique – testing the limits of our technological know-how, having never been done before with each mission presenting its own unique challenges. Building and deploying these systems on a mission-bymission basis often involves significant complexity and costs.<br/><br/>
-                                     ESA decided to embark on an ambitious challenge: to fly double the number of satellites it does currently by 2030, looking to technology to help it achieve this goal, with more rapid and automated software delivery, shared services, more efficient use of resources and lower costs of deployment.<br/><br/>
-                                     To meet this goal, ESA engaged Canonical to deliver the architecture, design and deployment of Kubernetes and Ceph for persistent storage. ESA also chose Canonical’s Ceph distribution to consolidate the various storage technologies in use. Canonical’s Kubernetes distribution allowed ESA’s Missions Operations Infrastructure – IT (MOI-IT) team to easily deploy its workloads, integrated with the existing IT infrastructure services, giving it a greater ability to support ESA missions and those of its partners, with automatic updates and deployment, 24/7 monitoring and reduced system complexity.<br/><br/>
-                                     Kubernetes was perfect for ESA’s next generation multi-mission platform at ESOC. This open source system helps organisations orchestrate their workloads more efficiently with automation. However, Kubernetes’ fast-moving and steep learning curve makes it more demanding to manage. To reduce the engineering workload and management requirements of this new system, ESA chose Canonical to manage its Kubernetes deployments, providing updates, monitoring and maintenance.<br/><br/>
-                                     The systems Canonical deployed and now manage for ESA mean that ESOC engineering teams can quickly and centrally manage and deploy mission-critical infrastructure and software, with more security, reliability, and control, and with less downtime and reduced risk of system failures."
-                      )
+  {{ text_template.text(
+      title="You can’t explore space without infrastructure.",
+      description="It’s not just expert personnel, satellites and spacecraft: it takes mountains of software, computer systems, data and networks to get these missions off the ground. These missions are often unique – testing the limits of our technological know-how, having never been done before with each mission presenting its own unique challenges. Building and deploying these systems on a mission-bymission basis often involves significant complexity and costs.<br/><br/>
```

Comment:
I wonder if there is a better way to denote different paragraphs. The double break `<br/><br/>` feels a little strange to me and might be too much to ask from content writers

Reply:
I implemented it as a list for now so content writers can input each paragraph as a list item. I tried doing a new paragraph but the spacing doesn't seem right so stuck with the double break in the macros for now. Let me know what you think!